packages/google_maps_flutter/lib/src/google_map.dart

```diff
@@ -88,6 +89,9 @@ class GoogleMap extends StatefulWidget {
   /// True if the map view should respond to scroll gestures.
   final bool scrollGesturesEnabled;
 
+  /// True if the map view should show zoom controls. Android only.
```

Comment:
Maybe state the default value is true.
Also explain a bit about what zoom control is, like: https://developers.google.com/android/reference/com/google/android/gms/maps/UiSettings.html#setZoomControlsEnabled(boolean)

Reply:
This has been done but not the documentation is doubled on getter & setter. 😐